### PR TITLE
Add sbt-project-graph, lexicographically order projects

### PR DESCRIPTION
**Pull request by edmundnoble**
I saw that I would have to edit the "dependency tree" in the comments of build.sbt when I added a benchmarks project for mimir, which seemed an incredibly daunting task. Then I thought, how many projects have we not added because of that feeling of dread? Then I thought, how much of this tree is even accurate? Why not ask SBT to do it? So I've added a plugin which does this exactly, and purged our "tree comment".

### CODE REVIEW

File: project/plugins.sbt [hunk 1]
```diff
@@ -1,9 +1,12 @@
...
-addSbtPlugin("com.eed3si9n"          % "sbt-assembly"  % "0.14.5")
-addSbtPlugin("com.eed3si9n"          % "sbt-buildinfo" % "0.7.0")
-addSbtPlugin("io.get-coursier"       % "sbt-coursier"  % "1.1.0-M4")
-addSbtPlugin("com.slamdata"          % "sbt-slamdata"  % "1.2.0")
-addSbtPlugin("pl.project13.scala"    % "sbt-jmh"       % "0.2.27")
-addSbtPlugin("com.github.romanowski" % "hoarder"       % "1.0.2-RC2")
+addSbtPlugin("com.eed3si9n"          % "sbt-assembly"      % "0.14.5")
+addSbtPlugin("com.eed3si9n"          % "sbt-buildinfo"     % "0.7.0")
+addSbtPlugin("io.get-coursier"       % "sbt-coursier"      % "1.1.0-M4")
+addSbtPlugin("io.get-coursier"       % "sbt-coursier"      % "1.1.0-M4")
```
Comment (alissapajer) [inline project/plugins.sbt]: This is doubled up.

**Comment by alissapajer** [comment]
Since the plugin isn't a requirement of the build, I'd prefer to not include it here. Anyone who wants it can include it in their global plugins.

File: project/plugins.sbt (hunk 1 above)
Comment (edmundnoble) [reply project/plugins.sbt]: Fixed! :)

**Comment by edmundnoble** [comment]
@alissapajer Donezo, just removed the tree from the comments.

**Comment by alissapajer** [comment]
So, I've really enjoyed the 5 minutes I'd spend every few weeks editing the ASCII graph, and it was nearly correct. I was actually just looking at it today thinking about how I could fix the last bit that wasn't correct :)

But, I agree it requires maintenance which is entirely unnecessary. I just drew the graph with the plugin and it's actually pretty nice. So I'm :+1: to merge :)

**Comment by edmundnoble** [comment]
Alright awesome. 

I could totally see how it would be fun to do, heh.

**Comment by edmundnoble** [comment]
Here's a copy of what I got when generating it, for the curious:

image

**Comment by alissapajer** [comment]
Is there any way to omit `root` (and thereby omit all the dashed lines?). I didn't see one in the docs.

**Pull request merged**